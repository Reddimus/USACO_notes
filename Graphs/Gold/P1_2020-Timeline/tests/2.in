100 10000 200
1934 6330 676 2386 3201 5040 867 6824 1446 1867 4 2022 2214 2186 659 1979 2517 862 496 4355 6319 379 4089 5655 3946 2634 4174 6168 669 4838 7230 2955 2024 557 2214 2466 2061 3640 549 892 3984 5058 238 4327 771 6288 350 2970 816 5121 1994 1048 7799 1645 836 1566 1686 2163 3324 512 1306 181 2171 2715 1869 1076 308 230 2184 272 5147 4437 277 5135 6129 959 1452 1871 3832 78 609 1885 412 2931 1394 2386 4839 2059 954 3239 2451 437 370 7217 2071 2862 673 6742 2677 3878
99 91 2319
35 91 4440
80 33 830
22 72 2142
62 70 26
63 96 4561
59 72 4186
65 3 131
29 22 2438
18 43 1698
54 72 851
60 89 2189
14 78 2657
23 57 719
28 61 773
80 28 2496
62 93 1710
48 46 1514
63 48 3965
46 75 130
23 100 1
47 92 2918
68 63 1902
85 30 2836
17 6 918
4 59 2044
36 28 2612
5 9 1293
23 39 343
100 53 252
27 53 1856
65 30 1041
85 53 552
97 64 2097
51 73 2496
49 85 2954
34 96 3855
64 28 420
90 71 1486
4 20 468
3 36 868
92 27 3743
10 83 3977
55 42 1890
67 15 1213
29 88 2994
11 71 4955
89 32 692
43 99 159
41 38 2864
30 31 1284
77 88 2482
19 12 1548
59 50 1914
85 55 445
33 48 676
80 57 865
34 66 1214
26 28 14
43 53 3326
55 28 56
7 26 2675
11 24 5785
85 25 888
69 27 3675
99 39 379
65 86 3623
78 39 375
19 8 6485
15 52 509
66 94 3935
16 31 3082
19 55 4311
12 30 810
45 40 919
52 17 3947
34 76 32
60 39 2092
77 16 659
99 72 585
19 75 3209
26 44 816
25 27 2710
85 54 2569
23 8 815
48 71 1777
16 54 2925
89 77 641
90 84 251
70 36 2708
83 73 391
33 32 318
6 94 549
9 94 1777
83 55 563
66 24 312
14 95 1791
18 58 1498
62 18 293
12 1 198
49 18 675
67 66 2572
1 51 1237
16 78 1119
70 63 2101
65 33 71
97 72 360
65 51 3023
56 40 187
7 23 99
81 50 3299
3 36 113
36 2 5092
67 100 2020
15 79 6965
60 24 2839
67 77 642
11 59 1593
26 57 517
76 46 1578
23 84 1621
35 8 2253
67 36 2782
97 77 241
49 85 1749
96 53 2977
71 30 400
34 21 3811
50 82 3060
26 79 2310
73 31 539
70 80 43
51 58 999
41 100 2528
69 48 1633
67 63 888
100 37 1649
13 30 2183
33 83 1160
20 61 3226
83 46 1520
45 29 3616
88 79 755
65 29 1756
33 73 429
12 13 1118
7 16 5
44 87 992
70 77 3346
91 58 979
52 46 5899
80 43 878
52 41 3214
74 84 382
29 17 740
49 47 31
27 44 118
32 95 4261
62 76 2896
51 8 3244
10 35 644
58 8 41
42 2 277
52 7 1862
15 94 315
66 79 5083
36 48 3186
25 72 4187
48 13 142
71 31 445
15 33 499
88 37 1802
98 95 1538
94 74 1230
67 18 738
69 72 2811
9 98 3090
24 23 794
83 21 3548
4 64 57
41 90 248
35 21 4342
83 55 585
33 48 2399
78 94 270
77 84 1366
51 54 2214
76 20 1590
76 64 187
44 74 70
3 69 151
3 34 215
96 75 2774
35 36 135
96 73 103
90 23 2070
20 24 1686
99 92 11
89 99 931
88 46 2016
