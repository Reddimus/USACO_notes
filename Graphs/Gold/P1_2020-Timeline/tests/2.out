2242
8596
2000
2386
3201
6895
3892
8377
4494
1867
4
2044
6480
2186
1521
4277
5977
1491
496
4667
9392
6825
7562
6353
4658
6567
7508
6581
4387
8663
9947
3393
2024
2215
2511
3090
9421
8108
7905
1753
5244
8319
3189
7626
771
9397
847
6338
816
6344
4892
2030
9911
7202
6429
1566
8281
7930
4430
512
7893
181
2373
3264
1869
3429
308
230
2184
272
8115
8967
7388
8674
9708
3077
3618
5396
8877
315
609
9404
5844
9183
3770
5492
8618
7381
2701
5492
6951
3765
1891
7444
9122
6934
673
7584
3632
7772
